add how to run a sequencer docs (#2229)
OffchainLabs/arbitrum-docs: arbitrum-docs/run-arbitrum-node/partials/run-full-node/_optional-orbit-sequencer-compatible-cli-partial.mdx

<section class='small-table'>

| Flag                                                      | Description                                                                                                                                                                                                                                                                                     |
| --------------------------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
| `--execution.rpc.classic-redirect=<RPC>`                  | Redirects archive requests for pre-nitro blocks to this RPC of an Arbitrum Classic node with archive database. Only for Arbitrum One.                                                                                                                                                           |

> | `--http.api`                                              | Offered APIs over the HTTP-RPC interface. Default: `net,web3,eth,arb`. Add `debug` for tracing.                                                                                                                                                                                                 |

Author: Yes, it means open which api, like `--http.api ["net","web3","eth","arb"]`, how about we say it as `Which APIs need to be opened over the HTTP-RPC interface.`?

> | `--http.api`                                              | Offered APIs over the HTTP-RPC interface. Default: `net,web3,eth,arb`. Add `debug` for tracing.                                                                                                                                                                                                 |

Reviewer: Sounds good. I pushed the change.

| `--execution.sequencer.expected-surplus-soft-threshold`   | If the expected surplus is lower than this value, warnings are posted (default "default").                                                                                                                                                                                                                 |
| `--execution.sequencer.forwarder.connection-timeout`      | Total time to wait before cancelling connection (default 30s).                                                                                                                                                                                                                                             |
| `--execution.sequencer.forwarder.idle-connection-timeout` | Time until idle connections are closed (default 1m0s).                                                                                                                                                                                                                                                     |

> | `--execution.sequencer.forwarder.max-idle-connections`    | Maximum number of idle connections to keep open (default 100).                                                                                                                                                                                                                                             |

Reviewer: ```suggestion
| `--execution.sequencer.forwarder.max-idle-connections`    | Maximum number of idle connections to keep open (default `100`).                                                                                                                                                                                                                                             |
```

> | `--execution.sequencer.forwarder.redis-url`               | The Redis URL to recommend target via.                                                                                                                                                                                                                                                                     |

Reviewer: ```suggestion
| `--execution.sequencer.forwarder.redis-url`               | The recommended Redis URL to use as target.                                                                                                                                                                                                                                                                     |
```

| `--execution.sequencer.max-acceptable-timestamp-delta`    | Maximum acceptable time difference between the local time and the latest L1 block's timestamp (default 1h0m0s).                                                                                                                                                                                            |
| `--execution.sequencer.max-block-speed`                   | Minimum delay between blocks (sets a maximum speed of block production) (default 250ms).                                                                                                                                                                                                                   |
| `--execution.sequencer.max-revert-gas-reject`             | Maximum gas executed in a revert for the sequencer to reject the transaction instead of posting it (anti-DOS).                                                                                                                                                                                             |

> | `--execution.sequencer.max-tx-data-size`                  | Maximum transaction size the sequencer will accept (default 95000).                                                                                                                                                                                                                                        |

Reviewer: ```suggestion
| `--execution.sequencer.max-tx-data-size`                  | Maximum transaction size the sequencer will accept (default `95000`).                                                                                                                                                                                                                                        |
```

> | `--execution.sequencer.nonce-cache-size`                  | Size of the transaction sender nonce cache (default 1024).                                                                                                                                                                                                                                                 |

Reviewer: ```suggestion
| `--execution.sequencer.nonce-cache-size`                  | Size of the transaction sender nonce cache (default `1024`).                                                                                                                                                                                                                                                 |
```

> | `--execution.sequencer.nonce-failure-cache-expiry`        | Maximum time to wait for a predecessor before rejecting a transaction with nonce too high (default 1s).                                                                                                                                                                                                    |

Reviewer: ```suggestion
| `--execution.sequencer.nonce-failure-cache-expiry`        | Maximum time to wait for a predecessor before rejecting a transaction whose nonce is too high (default 1s).                                                                                                                                                                                                    |
```

> | `--execution.sequencer.nonce-failure-cache-size`          | Number of transactions with too high of a nonce to keep in memory while waiting for their predecessor (default 1024).                                                                                                                                                                                      |

Reviewer: ```suggestion
| `--execution.sequencer.nonce-failure-cache-size`          | Number of transactions whose nonce is too high to keep in memory while waiting for their predecessor (default `1024`).                                                                                                                                                                                      |
```

> | `--execution.sequencer.queue-size`                        | Size of the pending transaction queue (default 1024).                                                                                                                                                                                                                                                      |

Reviewer: ```suggestion
| `--execution.sequencer.queue-size`                        | Size of the pending transaction queue (default `1024`).                                                                                                                                                                                                                                                      |
```

> | `--execution.sequencer.queue-timeout`                     | Maximum time transaction can wait in a queue (default 12s).                                                                                                                                                                                                                                                |

Reviewer: ```suggestion
| `--execution.sequencer.queue-timeout`                     | Maximum time a transaction can wait in a queue (default 12s).                                                                                                                                                                                                                                                |
```

> | `--execution.sequencer.sender-whitelist`                  | Comma-separated allowlist of authorized senders (if empty, everyone is allowed).                                                                                                                                                                                                                           |

Reviewer: ```suggestion
| `--execution.sequencer.sender-whitelist`                  | Comma-separated allowlist of authorized senders (if empty, every sender is allowed).                                                                                                                                                                                                                           |
```

> | `--node.delayed-sequencer.finalize-distance`              | How many blocks in the past L1 block is considered final (ignored when using Merge finality) (default 20).                                                                                                                                                                                                 |

Reviewer: ```suggestion
| `--node.delayed-sequencer.finalize-distance`              | Number of blocks in the past L1 block for the transaction to be considered final. This value is ignored when using merge finality. Default: `20`.                                                                                                                                                                                                 | 
```
